sync/class.jetpack-sync-module-users.php

```diff
@@ -71,9 +71,8 @@ public function init_before_send() {
 	}
 
 	public function sanitize_user_and_expand( $user ) {
-		$user = $this->sanitize_user( $user );
-
-		return $this->add_to_user( $user );
+		$user = $this->add_to_user( $user );
+		return $this->sanitize_user( $user );
```

Comment:
I'm curious, is there a reason why you reversed the order here?

Reply:
In theory it doesn't really matter, but when we call the add_to_user method I didn't want to take away from the user object or change the user object too much. Since in add_to_user calls add the get_allowed_mime_types( $user) and get_user_local( ); methods. Which rely on the user object to be certain shape. 

The sanitize_user method takes away properties and changes that object. Which made sense to me to follow after.